gnosis/gp-swap-ui
# [421] Environment-aware Explorer URLs

Closes #421 

Checks the environment and uses `.env.production` urls + `chainId` to construct `explorer` URLs correctly.

## QA Testing
:warning: _right now this can only best tested easily once merged in `develop` or `master` as the available URLs here won't trigger anything_

1. make orders and check the toast notification URLs are correct depending on what URL and chainId you are on
  ```bash
cow-trade.dev = https://protocol-explorer.dev.gnosisdev.com
cow-trade.staging = https://protocol-explorer.staging.gnosisdev.com
cow.trade = https://gnosis-protocol.io
```
 - can be checked in `Networks` tab as well
